storeutils/entity_id.go

```diff
@@ -98,3 +92,8 @@ func (id *EntityID) UnmarshalJSON(data []byte) error {
 func (id EntityID) MarshalJSON() ([]byte, error) {
 	return sdk.Uint(id).MarshalJSON()
 }
+
+func sdkUint2Big(in sdk.Uint) *big.Int {
```

Comment:
This can be removed in favor of `github.com/cosmos/cosmos-sdk/types.Uint.BigInt()`